lib/diagnostic.rb

```diff
@@ -108,10 +110,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = { 'favorite_number': 7, 'first_name': 'Catherine' }
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = "#{person_hash[:'first_name']} says hi!"
```

Comment:
Almost! To set a default return value we would say `person.default = "#{person_hash[:first_name]} says hi!"`
We actually don't need "" around first_name!